### PR TITLE
DOC: Add example for read_csv with nullable dtype

### PR DESCRIPTION
**Pull request by Idilismiguzel**
The first issue in the link: https://github.com/noatamir/pyladies-berlin-sprints/issues/4

### CODE REVIEW

**Comment by Idilismiguzel** [comment]
@noatamir, @phofl @mroeschke

File: doc/source/user_guide/io.rst [hunk 1]
```diff
@@ -464,6 +464,15 @@ worth trying.
 
    os.remove("foo.csv")
 
+.. ipython:: python
```
Comment (phofl) [inline doc/source/user_guide/io.rst:467]: Could you maybe add a small comment before the python block like: 

Setting ``use_nullable_dtypes=True`` will result in nullable dtypes for every column.

Between line 465 and 467

Comment (Idilismiguzel) [reply doc/source/user_guide/io.rst:469]: Sure, done!

**Review by mroeschke — APPROVED** [review]
Nice work. Looks good to me given the CI checks pass

**Comment by WillAyd** [comment]
Thanks @Idilismiguzel

**Pull request merged**